nf_core/lint.py

```diff
@@ -757,39 +737,10 @@ def check_conda_dockerfile(self):
 
         difference = set(expected_strings) - set(self.dockerfile)
         if not difference:
-            self.passed.append((10, "Found all expected strings in Dockerfile file"))
-        else:
-            for missing in difference:
-                self.failed.append((10, "Could not find Dockerfile file string: {}".format(missing)))
-
-    def check_conda_singularityfile(self):
-        """Checks the Singularity build file.
-
-        Checks that:
-            * a name is given and is consistent with the pipeline name
-            * dependency versions are pinned
-            * dependency versions are the latest available
-        """
-        if 'environment.yml' not in self.files or len(self.singularityfile) == 0:
-            return
-
-        expected_strings = [
-            'From:nfcore/base',
-            'Bootstrap:docker',
-            'VERSION {}'.format(self.config.get('manifest.version', '').strip(' \'"')),
-            'PATH=/opt/conda/envs/{}/bin:$PATH'.format(self.conda_config['name']),
-            'export PATH',
-            'environment.yml /',
-            '/opt/conda/bin/conda env create -f /environment.yml',
-            '/opt/conda/bin/conda clean -a',
-        ]
-
-        difference = set(expected_strings) - set(self.singularityfile)
-        if not difference:
-            self.passed.append((10, "Found all expected strings in Singularity file"))
+            self.passed.append((9, "Found all expected strings in Dockerfile file"))
         else:
             for missing in difference:
-                self.failed.append((10, "Could not find Singularity file string: {}".format(missing)))
+                self.failed.append((9, "Could not find Dockerfile file string: {}".format(missing)))
```

Comment:
I'm pretty sure that the Dockerfile test is still number 10... right?

Reply:
No, it's 9. https://github.com/matq007/tools/blob/master/docs/lint_errors.md

Reply:
Aha, nice! So this was an error in the previous code? Good spot 👍